### PR TITLE
Update docfx.json

### PR DESCRIPTION
**Pull request by lindalu-MSFT**
added "ms.suite": "office",

### CODE REVIEW

**Comment by PRMerger7** [comment]
@lindalu-MSFT : Thanks for your contribution! The author, @, has been notified to review your proposed change.

File: docs/docfx.json [hunk 1]
```diff
@@ -82,6 +82,7 @@
       "ms.topic": "conceptual",
       "audience": "developer",
       "defaultDevLang": "csharp",
+      "ms.suite": "office",
```
Comment (nschonni) [inline docs/docfx.json]: Why would the Visual Studio docs be in the Office suite?

**Comment by ktoliver** [comment]
Hi @lindalu-MSFT - Microsoft authors and contributors need to make content contributions through the private repository so the content can be staged and validated by the current validation rules. 

After you review staging and fix all validation issues, you can add the sign-off comment to let PR reviewers know that the updates are ready to be merged. 

For more information, see [Should I work in a private or a public repo?](https://review.docs.microsoft.com/en-us/help/contribute/contribute-authors-in-public-repo?branch=master) 

If you need help getting set up to work in the private repo, see [Set up Git repository locally for documentation](https://review.docs.microsoft.com/en-us/help/contribute/contribute-get-started-setup-local?branch=master).

Would you create a pull request in the private repo to make these updates? We'll close this PR. Thank you.

**Comment by lindalu-MSFT** [comment]
This ms.suite tag is actually for the Microsoft Office Developer platform. We are tracking metrics targeting developers who use VSTO. Knowing which docsets (and topics) Office Developers use helps us improve the office dev experience! Thank you, lindalu

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by lindalu-MSFT** [comment]
P.S. yes, I intended to do this PR in a private repo then merge to staging, sorry about that! I will redo the PR.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Pull request closed without merging**